.neutrinorc.js

```diff
@@ -89,7 +89,7 @@ if (process.env.REACT_APP_LIB_MODE) {
       react({
         html: {
           title: 'mobility-toolbox-js',
-          favicon: 'src/doc/img/favicon.png',
+          favicon: 'src/doc/img/favico.ico',
```

Comment:
I have tried to directly use the favicon from the geops-ui (see https://gitlab.geops.de/geops/geops-ui/-/blob/master/src/lib.js#L9). But when I require `@geops/geops-ui` in `.neutrinorc.js`, I get the error

`Error: Cannot find module '@geops/geops-ui'`

on `yarn start`.

(Even if it would work, I'm not sure if it will be accepted at the neutrino configuration in this line here.)

@edellucien, do you have an idea?

Reply:
require expect to load an index.js

you can require this instead:
const lib = require('@geops/geops-ui/lib.js');

But since this package is not builded and transpiled you will get:
SyntaxError: Unexpected token 'export'

Is it planned to compile'@geops/geops-ui'?

Reply:
Only buble is used - similarly to react-spatial. And the import in a module works fine as lib.js is referenced as module in package.json.

Reply:
@edellucien I suggest, to just merge this. So, the favicon has a better quality. In a next step, we could change the geops-ui API and implement a compile process.